### PR TITLE
Ft Reset password

### PR DESCRIPTION
**Pull request by Azutech**
#ft/reset-password, Implement user reset password
PR Description Template
 #### Description #### Type of change
 Please select the relevant option - [ ] Bug fix (non-breaking changes which fixes an issue)
 - [X] New feature (non-breaking changes which adds functionality)
 - [ ] Breaking change (fix or feature that could cause existing functionality to not work as expected)
 - [ ] This change requires a documentation update
 - [X] Chore #### How Has This Been Tested?
 please describe the test that you ran to verify your changes 
 - [ ] End to end
 - [ ] Integration #### Checklist: - [X] My code follows the style guidelines of this project
 - [X] I have linted my code prior to submission
 - [ ] I have made corresponding changes to the documentation
 - [X] My changes generates no new warnings
 - [ ] I have added tests that prove my fix is effective or that my feature works
 - [X] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by Azutech** [comment]
please kindly review

File: src/Controllers/Auth/authController.ts [hunk 1]
```diff
@@ -14,3 +15,8 @@ export const verifyUserEmail = async (req: Request, res: Response) => {
 export const authenticate = async (req: Request, res: Response) => {
     await loginService(req, res);
 };
+
+export const resetPass = async (req: Request, res: Response) => {
+    await resetPasswordService(req, res);
```
Comment (forestbaba) [inline src/Controllers/Auth/authController.ts:20]: What are you trying to do here??

File: src/services/Auth/index.ts [hunk 2]
```diff
@@ -82,3 +82,39 @@ export const loginService = async (req: Request, res: Response) => {
         res.status(400).json(error);
     }
 };
+
+export const resetPasswordService = async (req: Request, res: Response) => {
+ 
+    try {
+      const user = await User.findOne(req.body.userId);
```
Comment (forestbaba) [inline src/services/Auth/index.ts:89]: Why are you calling a model instead of a repository?

File: src/Controllers/Auth/authController.ts (hunk 1 above)
Comment (Azutech) [reply src/Controllers/Auth/authController.ts:20]: the resetPasswordService was exported from the Auth folder to the controller folder, resetpass initiates it

File: src/services/Auth/index.ts (hunk 2 above)
Comment (Azutech) [reply src/services/Auth/index.ts:89]: a different model was built for tokens so I referenced it to users model

**Pull request closed without merging**